<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SecDialog</class>
 <widget class="QDialog" name="SecDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1069</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="sendButton">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>420</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>430</y>
     <width>171</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="clock_label_2">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Clock</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="dateTime_label_2">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>DateTime</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTextBrowser" name="messageBrowser">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>40</y>
     <width>611</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="messageBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>420</y>
     <width>511</width>
     <height>70</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="users_text">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>171</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
